CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ with the built-in client to an already running nREPL server.
 #### Bugs fixed
 
 * [#38](https://github.com/nrepl/nREPL/issues/38): Remove extra newline in REPL output.
+* [#10](https://github.com/nrepl/nREPL/issues/10): Bind *1, *2, *3 and *e in cloned session.
```

Comment:
This should go to the `master` section. (you need to to add there "Bugs fixed" section, as there are no entries for `master` yet).